### ISSUE TITLE
Prepare authentication when SSH agent is not running

### ISSUE THREAD

**Issue by siloekse** [uitml/springfield#23]
I came across an edgecase when the SSH agent is not running when prepare-authentication.sh is executed. The key/pub files are created, but they will of course not be added to the agent. The files will only be added to the agent if they don't exist beforehand, so in this case the user will have to add them manually after starting the SSH agent and run prepare-authentication.sh again to configure k8s.

**Comment by thomasjo** [comment]
Yeah, good observation. I never found a truly reliable way to automate this, since it's highly dependent upon OS/distribution. Most user-friendly Linux distributions start ssh-agent during login, either using a systemd service or a short blurb in e.g. `/etc/bashrc` or similar.

Maybe we should detect if ssh-agent is running, and present the user with a helpful warning message if it isn't?

**Comment by siloekse** [comment]
Yeah. A check/warning message with instructions on how to proceed should be sufficient. The agent might be stopped on purpose.
